src/page-modules/departures/details/details.module.css

```diff
@@ -0,0 +1,100 @@
+.container {
+  margin: 0 auto;
+  max-width: var(--maxPageWidth);
+  padding: var(--spacings-xLarge);
+
+  display: grid;
+  grid-template-columns: 2fr 3fr;
+  grid-template-areas:
+    'header header'
+    'serviceJourney map';
+  gap: var(--spacings-small);
+}
+@media (max-width: 650px) {
+  .container {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      'header'
+      'serviceJourney'
+      'map';
+  }
+}
+.headerContainer {
+  grid-area: header;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacings-large);
+}
+.mapContainer {
+  grid-area: map;
+  max-height: 37.5rem;
+  width: 100%;
+  margin: 0 auto;
+}
+.serviceJourneyContainer {
+  grid-area: serviceJourney;
+}
+@media (max-width: 650px) {
+  .serviceJourneyContainer {
+    margin: 0 auto;
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-large);
+}
+
+.realtimeText {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-small);
+  font-size: 0.875rem;
+  color: var(--text-colors-secondary);
+}
+
+.callRows {
+  margin-top: var(--spacings-large);
+}
+.callRows__container {
+  padding: var(--spacings-medium);
+}
+.rowContainer {
+  position: relative;
+  --labelWidth: 5rem;
```

Comment:
I see I've been using kebab case for variables, but probably should have used camel case as it is here. We should probably be consistent with it. The variables in `globals.css` are camel cased so that's probably the way to go.

Reply:
I agree that we should be consistent! I can change this if you prefer kebab case. I think I'm a bit all over the place regarding this...

Reply:
No strong preference either. @mikaelbr can decide.